docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm

```diff
@@ -72,7 +73,11 @@ <h2 id="manual-constructor">How to Create Section Changer</h2>
          &lt;/section&gt;
       &lt;/div&gt;
    &lt;/div&gt;
-&lt;/div&gt;</pre><pre class="prettyprint lang-js" style="border:0px; margin:0px">(function()
+&lt;/div&gt;
+</pre>
+<p>JS code:</p>
+<pre class="prettyprint">
+(function()
 {
    var page = document.getElementById(&quot;hasSectionchangerPage&quot;),
        element = document.getElementById(&quot;sectionchanger&quot;),
```

Comment:
Would you please check this \</pre>\</pre> on line 103?
It looks like it should be \</pre> to me.

Reply:
Fixed